var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -273,56 +273,64 @@ def libs(self):
         spec = self.spec
 
+        cm_label = ""
+        if "@7.5:" in spec:
+            cm_label = "SIRIUS_"
+
         args = [
-            self.define_from_variant("USE_OPENMP", "openmp"),
-            self.define_from_variant("USE_ELPA", "elpa"),
-            self.define_from_variant("USE_MAGMA", "magma"),
-            self.define_from_variant("USE_NLCGLIB", "nlcglib"),
-            self.define_from_variant("USE_VDWXC", "vdwxc"),
-            self.define_from_variant("USE_MEMORY_POOL", "memory_pool"),
-            self.define_from_variant("USE_SCALAPACK", "scalapack"),
-            self.define_from_variant("CREATE_FORTRAN_BINDINGS", "fortran"),
-            self.define_from_variant("CREATE_PYTHON_MODULE", "python"),
-            self.define_from_variant("USE_CUDA", "cuda"),
-            self.define_from_variant("USE_ROCM", "rocm"),
-            self.define_from_variant("BUILD_TESTING", "tests"),
-            self.define_from_variant("BUILD_APPS", "apps"),
-            self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
-            self.define_from_variant("USE_FP32", "single_precision"),
-            self.define_from_variant("USE_PROFILER", "profiler"),
-            self.define_from_variant("USE_WANNIER90", "wannier90"),
+            self.define_from_variant(cm_label + "USE_OPENMP", "openmp"),
+            self.define_from_variant(cm_label + "USE_ELPA", "elpa"),
+            self.define_from_variant(cm_label + "USE_MAGMA", "magma"),
+            self.define_from_variant(cm_label + "USE_NLCGLIB", "nlcglib"),
+            self.define_from_variant(cm_label + "USE_VDWXC", "vdwxc"),
+            self.define_from_variant(cm_label + "USE_MEMORY_POOL", "memory_pool"),
+            self.define_from_variant(cm_label + "USE_SCALAPACK", "scalapack"),
+            self.define_from_variant(cm_label + "CREATE_FORTRAN_BINDINGS", "fortran"),
+            self.define_from_variant(cm_label + "CREATE_PYTHON_MODULE", "python"),
+            self.define_from_variant(cm_label + "USE_CUDA", "cuda"),
+            self.define_from_variant(cm_label + "USE_ROCM", "rocm"),
+            self.define_from_variant(cm_label + "BUILD_TESTING", "tests"),
+            self.define_from_variant(cm_label + "BUILD_APPS", "apps"),
+            self.define_from_variant(cm_label + "BUILD_SHARED_LIBS", "shared"),
+            self.define_from_variant(cm_label + "USE_FP32", "single_precision"),
+            self.define_from_variant(cm_label + "USE_PROFILER", "profiler"),
+            self.define_from_variant(cm_label + "USE_WANNIER90", "wannier90"),
         ]
 
         lapack = spec["lapack"]
         blas = spec["blas"]
 
         args.extend(
             [
-                self.define("LAPACK_FOUND", "true"),
-                self.define("LAPACK_LIBRARIES", lapack.libs.joined(";")),
-                self.define("BLAS_FOUND", "true"),
-                self.define("BLAS_LIBRARIES", blas.libs.joined(";")),
+                self.define(cm_label + "LAPACK_FOUND", "true"),
```

Comment:
Aren't these from `find_package(BLAS)`?

Reply:
No. SIRIUS does not use findBLAS at all.

Reply:
I will fix this

Reply:
I reverted my changes after realizing that we do use the findLapack.cmake provided by cmake.